### ISSUE TITLE
Remove celo-extension-wallet

### ISSUE THREAD

**Issue by jmrossy** [celo-org/react-celo#349]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Consider removing the celo-extension-wallet option
It haven't been updated since June 2021. It's dead, better to not direct users there anymore. Especially since Metamask works well with Celo these days.

### Steps to reproduce

-

### Expected behavior

-

### Where is the problem happening?

Both

### Environment

-

### What version of `react-celo` were you using? (This can be found in the list of wallets in the "Connect" modal)

latest

**Comment by nicolasbrugneaux** [comment]
Agreed, I'm going to remove it from defaults and deprecate it but not remove it yet. We'll remove it next major release.
